integration-cli/requirements.go

```diff
@@ -37,7 +38,18 @@ var (
 	}
 	Network = TestRequirement{
 		func() bool {
-			resp, err := http.Get("http://hub.docker.com")
+			// Set a timeout on the GET at 15s
+			var timeout = time.Duration(15 * time.Second)
+			var url = "https://hub.docker.com"
+
+			client := http.Client{
+				Timeout: timeout,
+			}
+
+			resp, err := client.Get(url)
+			if err != nil && strings.Contains(err.Error(), "use of closed network connection") {
+				panic(fmt.Sprintf("Timeout for GET request on %s", url))
```

Comment:
It doesn't really have to. The thing is, with TestRequirement, if there is an error, the test will be skipped. So in case of timeout, if I don't panic the test is skipped, which makes it harder to detect I think